libs/ui/src/reports/precinct_scanner_ballot_count_report.tsx

```diff
@@ -95,7 +95,8 @@ export function PrecinctScannerBallotCountReport({
               <dt>Time {getPollsTransitionActionPastTense(pollsTransition)}</dt>
               <dd>
                 {formatFullDateTimeZone(
-                  DateTime.fromMillis(pollsTransitionedTime)
+                  DateTime.fromMillis(pollsTransitionedTime),
+                  { includeSeconds: true }
```

Comment:
I know that this is existing logic prior to your PR but do you recall why this date/time includes day of the week while others do not? Wondering if it's worth making consistent

Reply:
I think that folks just included weekday when there was plenty of space and removed it when space was tight. This is a in a different location, in the body of the ballot count report rather than the header. I'm less concerned about consistency in different locations on reports, wanted to be consistent across all report headers. And I don't want to update snapshots. So leaving this for now.

Reply:
Gotcha, totally fine by me